assets/js/settings/shared/index.js

```diff
@@ -66,3 +67,40 @@ export { compareVersions, getSetting };
  * @return {string} Full admin URL.
  */
 export const getAdminLink = ( path ) => getSetting( 'adminUrl' ) + path;
+
+addFilter(
+	'woocommerce_admin_analytics_settings',
+	'woocommerce-admin',
```

Comment:
I believe this is meant to be the namespace of your plugin, not the plugin applying the filter ([`doAction()`](https://github.com/WordPress/gutenberg/tree/master/packages/hooks#api-usage) doesn't take a namespace argument).